### ISSUE TITLE
TypeInitializationException for RocksDbSharp.Native

### ISSUE THREAD

**Issue by ixje** [neo-project/neo-express#247]
I followed the instructions from the readme to install the dependencies (for Ubuntu 22.04)
> sudo apt install libsnappy-dev libc6-dev librocksdb-dev -y

```shell
erik@erik-linux:~/$ sudo apt install libsnappy-dev libc6-dev librocksdb-dev -y
[sudo] password for erik: 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
libsnappy-dev is already the newest version (1.1.8-1build3).
librocksdb-dev is already the newest version (6.11.4-3).
libc6-dev is already the newest version (2.35-0ubuntu3.1).
The following packages were automatically installed and are no longer required:
  dotnet-apphost-pack-6.0 dotnet-host dotnet-hostfxr-6.0 dotnet-runtime-6.0 dotnet-targeting-pack-6.0
  liblttng-ust-ctl4 liblttng-ust0 libunwind-13 liburcu6 netstandard-targeting-pack-2.1
Use 'sudo apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 3 not upgraded.
```
However actual usage of `neoxp` fails
```
erik@erik-linux:~/$ neoxp run -i default.neo-express 
System.TypeInitializationException: The type initializer for 'RocksDbSharp.Native' threw an exception.
```

I tried both global dotnet tool install and building it from source but the same error happens. For the from source version I can see the following files available
```
/bin/debug/net6.0/RocksDbSharp.dll
/bin/debug/net6.0/runtimes/linux-x64/native/librocksdb.so
```
So everything seems to be there, yet the error suggests it's not. I tried building https://github.com/warrenfalk/rocksdb-sharp manually and overwriting the above files with my locally build, still failing with the same error.

RocksDB works if I use `rocksdb-tools`
```
erik@erik-linux:~/$ ldb --db=/tmp/ --create_if_missing put firstname erik
OK
erik@erik-linux:~/$ ldb --db=/tmp get firstname
erik
```
This suggests to me that this is some dotnet package related issue
```
erik@erik-linux:~/$ dotnet --info
.NET SDK (reflecting any global.json):
 Version:   6.0.400
 Commit:    7771abd614

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  22.04
 OS Platform: Linux
 RID:         ubuntu.22.04-x64
 Base Path:   /home/erik/.dotnet/sdk/6.0.400/

global.json file:
  Not found

Host:
  Version:      6.0.8
  Architecture: x64
  Commit:       55fb7ef977

.NET SDKs installed:
  6.0.400 [/home/erik/.dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.8 [/home/erik/.dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.8 [/home/erik/.dotnet/shared/Microsoft.NETCore.App]
```

**Comment by devhawk** [comment]
we will update rocks db sharp for 3.5 release

**Comment by devhawk** [comment]
Resolved with 3.5 blockchain toolkit

**Comment by ixje** [comment]
For ARM based OSX users using homebrew to install; make sure to have homebrew installed under `/opt/` not under `/usr/local/` or you'll get this error as well ([ref](https://github.com/Homebrew/brew/pull/9117))
